notebooks/tel_and_site/subsys_req_ver/m2/SITCOMTN-096_M2_outer_control_closed_loop.ipynb

```diff
@@ -0,0 +1,1318 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #110.        fig.savefig(f"./plots/Time_vs_Forces_and_Step_on_A" f"{i+1}.png")
</pre><p>You can join the two stings into one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>